test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -153,22 +152,22 @@ public void TestSaveImages()
             {
```

Comment:
Should we add unit tests for the MLImage type?  I would expect us to validate public API, exception behavior, and lifetime.

Reply:
I added the unit tests covering MLImage class.